### DISCUSSION TITLE
serverpod default app not runing

### DISCUSSION THREAD

**Discussion by ElsimatAmir** [serverpod/serverpod#1349] [Q&A] [answered]
![image](https://github.com/serverpod/serverpod/assets/123176613/c07b8b71-bd70-4ddc-a9f9-f719f1de55f8)

just created a new app am runed the tables script and still having this issue

**Comment by Isakdl** [comment, marked as answer]
You running on windows? The tables are not created in the database. There should be a`.bat` script in the root folder that will help you set up the database the first time you run the app.

You can also checkout this answer for how to setup the tables manually without the script: https://stackoverflow.com/questions/76180598/serverpod-sql-error-when-starting-a-clean-project

**Reply by ElsimatAmir** [reply]
thanks yha i didit from begining not its worked
